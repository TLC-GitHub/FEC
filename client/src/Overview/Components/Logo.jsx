import React, {useState} from 'react';
import styled from 'styled-components';

const Logo = () => {
  const [data, setData] = useState('');
  const getData = () => setData(data);



  return (
   <h1>lelelime</h1>
  );
}

export default Logo;