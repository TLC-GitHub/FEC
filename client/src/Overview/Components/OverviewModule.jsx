import React, {useState} from 'react';

const OverviewModule = () => {
  const [data, setData] = useState('');
  const getData = () => setData(data);

  return (
    <div></div>
  );
}

export default OverviewModule;