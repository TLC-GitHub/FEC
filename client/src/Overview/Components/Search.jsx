import React, {useState} from 'react';
import axios from 'axios';

const Search = () => {
  const [search, setSearch] = useState('');

  const handleChange = (event) => {
    event.preventDefault();
    setSearch(event.target.value);
  }

  return (
      <form>
        <label>Search Products</label>
        <input type="text" name="product-search" onChange={handleChange}/>
      </form>

  );
}

export default Search;