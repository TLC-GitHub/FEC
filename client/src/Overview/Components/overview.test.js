import React from 'react';
import renderer from 'react-test-renderer';
import OverviewModule from './OverviewModule.jsx';
import Logo from './Logo.jsx';
import Search from './Search.jsx';

describe('overview module', () => {
  it('renders the OverviewModule component to the DOM', () => {
    const component = renderer.create(<OverviewModule />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders the Logo component to the DOM', () => {
    const component = renderer.create(<Logo />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders the Search component to the DOM', () => {
    const component = renderer.create(<Search />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});






