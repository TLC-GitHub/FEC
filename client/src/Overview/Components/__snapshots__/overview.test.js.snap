// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview module Logo component renders the Logo component to the DOM 1`] = `
<h1
  className="sc-bczRLJ fknBqq"
>
  Stylish Skin Coverings
</h1>
`;

exports[`Overview module OverviewModule component renders the OverviewModule component to the DOM 1`] = `<div />`;

exports[`Overview module Search component renders the Search component to the DOM 1`] = `
<form>
  <label>
    Search Products
  </label>
  <input
    name="product-search"
    onChange={[Function]}
    type="text"
  />
</form>
`;

exports[`overview module renders the Logo component to the DOM 1`] = `
<h1
  className="sc-bczRLJ eFcBDS"
>
  Stylish Skin Coverings
</h1>
`;

exports[`overview module renders the OverviewModule component to the DOM 1`] = `<div />`;

exports[`overview module renders the Search component to the DOM 1`] = `
<form>
  <label>
    Search Products
  </label>
  <input
    name="product-search"
    onChange={[Function]}
    type="text"
  />
</form>
`;

exports[`renders the Logo component to the DOM 1`] = `
<h1
  className="sc-bczRLJ fknBqq"
>
  Stylish Skin Coverings
</h1>
`;

exports[`renders the OverviewModule component to the DOM 1`] = `<div />`;

exports[`renders the Search component to the DOM 1`] = `
<form>
  <label>
    Search Products
  </label>
  <input
    name="product-search"
    onChange={[Function]}
    type="text"
  />
</form>
`;

exports[`renders the component to the DOM 1`] = `<div />`;
